lib/medium-admin.php

```diff
@@ -764,13 +765,26 @@ public static function cross_post($post, $medium_post, $medium_user) {
       $path = "/v1/users/{$medium_user->id}/posts";
     }
 
-
     return self::_medium_request("POST", $path, $medium_user->token, $data, array(
       "Content-Type" => "application/json"
     ));
   }
 
   /**
+   * Creates a post claim for the original author of a post.
+   */
+  public static function create_post_claim(Medium_Post $medium_post, Medium_User $medium_user, $user) {
+    $body = array(
+      "md5" => md5($user->user_email)
```

Comment:
We should probably downcase the email to make the comparison case-insensitive across WP/Medium